src/Servers/Kestrel/Core/test/SlabMemoryPoolTests.cs

```diff
@@ -26,4 +26,4 @@ public void DisposeWithActiveBlocksWorks()
             memoryPool.Dispose();
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Nit: I normally don't complain about BOM changes and newlines at the end of files, but if the file is otherwise untouched, leave it.